<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>707</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Расчет распределения фонда оплаты труда</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="shtat_tab">
     <attribute name="title">
      <string>Работа со штатным расписанием</string>
     </attribute>
     <widget class="QGroupBox" name="personel_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>461</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Данные сотрудника</string>
      </property>
      <widget class="QLabel" name="department">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Подразделение</string>
       </property>
      </widget>
      <widget class="QLabel" name="position">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Наименование должности</string>
       </property>
      </widget>
      <widget class="QLabel" name="fio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО работника</string>
       </property>
      </widget>
      <widget class="QLabel" name="tarif">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Оклад (тариф)</string>
       </property>
      </widget>
      <widget class="QLabel" name="salary">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Сумма к расчету</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pos_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="fio_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tarif_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="salary_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="pos_count_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="pos_count">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ставка</string>
       </property>
      </widget>
      <widget class="QComboBox" name="department_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="category">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Категория сотрудника</string>
      </property>
      <widget class="QRadioButton" name="na">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Руководитель</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="tt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Технолог</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ti">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>281</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Повременьщик</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="ws">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Вспомогательный рабочий</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="np">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Специалист</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="sd">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Сдельщик</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="history">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>471</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>История по позиции</string>
      </property>
      <widget class="QTextEdit" name="history_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>451</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Position_Condition">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>321</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Состояние позиции</string>
      </property>
      <widget class="QLabel" name="Salary_do">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Оклад замещающего работника:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="do_salary_input">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="do_y_n">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>159</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>д/о работника</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="vacancy">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Вакансия</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="search_number">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>500</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Позиция : 0 из 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="search_button">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>540</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск в штатном расписании</string>
      </property>
     </widget>
     <widget class="QPushButton" name="left_button_max">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>540</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right_button_max">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>540</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>590</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_position_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Новая позиция</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>590</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить позицию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_form_buton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>540</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить форму</string>
      </property>
     </widget>
     <widget class="QPushButton" name="left_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>540</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>540</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="work_shtat_button">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>590</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Работа со штатным расписанием</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_shtat_table_button">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>540</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Выгрузить ШР для расчета отклонений</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_personel_button">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>490</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Выгрузить численность</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fot_tab">
     <attribute name="title">
      <string>Работа с ФОТ</string>
     </attribute>
     <widget class="QTableWidget" name="stimul_table">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>761</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Данные для расчета</string>
      </property>
      <widget class="QLabel" name="fot_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ФОТ к распределению:</string>
       </property>
      </widget>
      <widget class="QLabel" name="file_otkl_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Файл отклонений:</string>
       </property>
      </widget>
      <widget class="QLabel" name="month_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Месяц:</string>
       </property>
      </widget>
      <widget class="QLabel" name="work_days_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество рабочих дней:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="month">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="stimul_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Коэффициент стимула:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="file_input">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="fot_input">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="workday_input">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="stimul_input">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="file_button">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="use_all_fot">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Использовать весь ФОТ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="count_button">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>220</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="message_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>441</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="save_stimul_table_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Выгрузить в файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="print_sheets_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>580</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Напечатать листочки</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
